packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -78,7 +78,6 @@ export const editorTextBoxButtonStyle: IStyle = {
  */
 export const attachmentUploadCardsStyles = mergeStyles({
   margin: '0 0.25rem 0.25rem 0.25rem',
-  maxHeight: '12.5rem',
```

Comment:
it's to prevent attachment being cutting off when you have more than 4 attachments in sendbox, it's a one line fix, i can move this to different PR if you would like